CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,9 @@
+<!--
+    SPDX-FileCopyrightText: 2014 Coraline Ada Ehmke
```

Comment:
is it normal that it is not 2022-2024 SeisSol Group?

Reply:
Effectively, we've just copied the file from there; didn't we? Honestly, I'm a bit uncertain myself; but I don't see any contribution from our side.

We can change it to `SeisSol Group` etc.; but I'm sure if we have a "copyright" there. The original CoC we definitely released under CC-BY 4.0 though.

(if we want to discuss these issues for thoroughly; I can also remove the statement for now from this file)

Reply:
Oh wait, I stand corrected. We _did_ modify the file. I'll add us. Or should we adjust the file even more?

Reply:
Ok; I've adjusted the header as you've suggested and only left the SeisSol Group—the attribution at the bottom should be good. I guess otherwise it might also give off the "wrong" impression.